capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Implementation of SearchResource Interface */
+public class SearchResourceImpl implements SearchResource {
+
+  public List<SearchData> getSearchSuggestions(List<String> collegeNames) {
```

Comment:
Method implementation should match interface
```suggestion
  @Override
  public List<SearchData> getSearchSuggestions(String collegeNameQuery) {
```